source/manual/setting-up-new-rails-app.html.md

```diff
@@ -172,88 +209,94 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+Information about how to test the app.
 
-- **Word**: definition of word, and how it's used in the code
+## Further documentation
 
-## Technical documentation
+More information on how the app works.
 
-Write a single paragraph including a general technical overview of the app.
+## API documentation
 
-### Dependencies
+If applicable, include API reference information including the following sections:
 
-- [dependency]() - purpose
+- resources
+- endpoints and methods
+- parameters
+- example requests and responses
+- error codes
 
-### Running the application
+## Licence
 
-How to run the app
+Link to the [MIT License](LICENCE).
+```
 
-### Running the test suite
+See the example [account-api README.md](https://github.com/alphagov/account-api/blob/main/README.md) for more information.
 
-How to test the app
+See the [GOV.UK Writing API reference documentation](https://www.gov.uk/guidance/writing-api-reference-documentation) for more information on API references.
 
-## Licence
+## Prepare the Rails app to run in production
 
-[MIT License](LICENCE)
-```
+### Configure the Rails app for Jenkins
 
-## Puppet, DNS, Sentry and beyond
+1. Create a `Jenkinsfile` in your repo with the following content.
 
-* To prepare the servers for your app, follow the doc in [govuk-puppet]
-* To configure app deployments, follow the docs in [govuk-app-deployment]
-* To enable external DNS, follow the article in the [Developer Docs][dns]
-* Make a PR to add your app to [data/applications.yml][docs-applications] for these docs
-* Then ask 2nd line to run the task in [govuk-saas-config] to update [Sentry]
-* Add the application to the [Release] app (the create button is at the bottom)
-* Add the application to the [Deploy_App job][deploy-jenkins] in Jenkins (click Configure)
-* Run the [Deploy_App job][deploy-jenkins] using with `with_migrations` option to get started
+    ```
+    #!/usr/bin/env groovy
 
-When adding your application to `data/application.yml` - ensure your new repository has
-a description set in the "About" section on github. This will be used to describe what your app
-is on the [application listings](https://docs.publishing.service.gov.uk/#applications) and is also
-required to render an application page.
+    library("govuk")
 
-If you do not do this, you may see Middleman errors when the docs are building which reference
-attempts to modify a frozen empty string.
+    node {
+      govuk.buildProject()
+    }
+    ```
 
-## Configuring the app for Jenkins
+1. Add a Jenkins integration to the repo on GitHub. See the [GOV.UK developer documentation on automatically configuring a GitHub repo](auto-config) for more information.
 
-Create a `Jenkinsfile` in your repo with the following content.
+### Set up the Rails app in govuk-puppet
 
-```
-#!/usr/bin/env groovy
+Add the Rails app to the list of deployable apps in `govuk-puppet`.See the [`govuk-puppet` documentation on adding a new app to GOV.UK](govuk-puppet) for more information.
```

Comment:
There are more steps than just updating the deployable apps list, so to avoid repeating the puppet docs maybe say something higher level like just "Follow the govuk-puppet documentation on adding a new app to GOV.UK."